src/pages/RoomInvitePage.js

```diff
@@ -1,3 +1,5 @@
+import {parsePhoneNumber} from 'awesome-phonenumber';
```

Comment:
Lint failing on main. Came from this PR.
```suggestion
import {parsePhoneNumber} from '@libs/PhoneNumber';
```
And move this line below `import * as OptionsListUtils from '@libs/OptionsListUtils';`

Same applies to WorkspaceInvitePage.

@thienlnam how about adding this fix to #34311?

Reply:
Ah, I already got started in https://github.com/Expensify/App/pull/34312